runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.java

```diff
@@ -155,7 +155,7 @@
 
   // Disable this support for now until the Dataflow backend fully supports this option.
   @Description("Whether to allow dynamic pubsub destinations. Temporary option: will be removed.")
```

Comment:
Could we just remove the option instead?

Reply:
done